server.js

```diff
@@ -0,0 +1,8 @@
+import "dotenv/config";
+import app from "./src/app.js";
+
+const PORT = 3000;
+
+app.listen(PORT, () => {
+  console.log("Fala que eu tô te escutando!!");

```

Comment:
Kkkkk achei criativo a mensagem para avisar que a API tá rodando, mas que tal uma mensagem mais específica para indicar sobre o funcionamento da API? Algo como:

```suggestion
  console.log(`app running on port ${PORT)`);
```

Reply:
:(